cli/tests/testdata/fmt/invalid_data.out

```diff
@@ -0,0 +1,4 @@
+error: [WILDCARD] is not a valid UTF-8 file
+
+Caused by:
+    invalid data
```

Comment:
Looking at this error... I think we should do `map_err()` instead of `with_context()`. "caused by: invalid data" provides no value to the user here